src/plugin.js

```diff
@@ -138,6 +138,18 @@ class PlaylistMenuItem extends Component {
     const li = document.createElement('li');
     const item = this.options_.item;
 
+    if (typeof item.data === 'object' && item.data) {
+      const dataKeys = Object.keys(item.data);
+
+      if (dataKeys.length > 0) {
```

Comment:
You don't need this if, as `Object.keys` will return an empty array if there are no keys, and then the `forEach` will be skipped. Since there is nothing to loop over.